tests/rkt_tests.go

```diff
@@ -219,6 +219,19 @@ func createTempDirOrPanic(dirName string) string {
 	return tmpDir
 }
 
+// The createFileOrPanic creates an empty file within the given directory
+// with a specified name. Panics if file creation fails for any reason.
+func createFileOrPanic(dirName, fileName string) string {
+	name := filepath.Join(dirName, fileName)
+	file, err := os.OpenFile(name, os.O_RDWR|os.O_CREATE|os.O_EXCL, 0600)
```

Comment:
Right, it does not matter in this situation, I initially though about close-exec flag, but as this function used only in tests this is, probably, not that important as well. So as you proposed, I changed to ```os.Create``` function.